### DISCUSSION TITLE
Symfony 6 compatibility

### DISCUSSION THREAD

**Discussion by spackmat** [spatie/browsershot#588] [Ideas]
Analogue to https://github.com/spatie/image/issues/151 it seems to be safe to allow `symfony/process` version `^6.0` in this package, I copy the text over from there:

I recently upgraded an app to Symfony 6.0 that uses spatie/browsershot, which itself requires spatie/image. Both state compatibility with several versions of symfony/process, but not with version ^6.0 of it. I managed to fix this with the following require line in my app's composer.json:

"symfony/process": "6.0.0 as 5.4",

That works fine, so it should be safe to allow version ^6.0 of symfony/process in both spatie/image and spatie/browsershot.

Greets!

**Comment by freekmurze** [comment]
Feel free to send a PR for this! 👍
